src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -3,14 +3,18 @@
     <div class="card-pf-heading f8-card-heading">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="card-pf-heading-details f8-card-heading-details" default-level>
-          <a id="spacehome-codebases-add-space-button" class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-            <i class="pficon pficon-add-circle-o"></i>
-          </a>
-        </div>
-        <div class="card-pf-heading-details f8-card-heading-details" user-level>
+          <div *ngIf="userOwnsSpace">
             <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-                <i class="pficon pficon-add-circle-o"></i>
+              <i id="default-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
             </a>
+          </div>
```

Comment:
Lines 5-18.

The user-level directive shouldn't be located inside the default-level directive.

Other than the link ID, the code for this feature toggle appears to be the same. Can we just remove it?

```
<div class="card-pf-heading-details f8-card-heading-details" default-level>
          <div *ngIf="userOwnsSpace">
            <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
              <i id="default-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
            </a>
          </div>
          <div class="card-pf-heading-details f8-card-heading-details" user-level>
            <div *ngIf="userOwnsSpace">
              <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
                <i id="user-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
              </a>
            </div>
          </div>
</div>
```